src/cli/domain/init-template/scaffold.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const Spinner = require('cli-spinner').Spinner;
+const strings = require('../../../resources');
+
+module.exports = function scaffold(options) {
+  const {
+    componentName,
+    componentPath,
+    compilerPath,
+    callback,
+    compiler,
+    logger
+  } = options;
+  const compilerPackage = require(compilerPath + '/package.json');
+
+  try {
+    const scaffold = new Spinner(strings.messages.cli.startScaffold());
+
+    scaffold.start();
+    const baseComponentPath = path.join(compilerPath, 'scaffold');
+    const baseComponentFiles = path.join(baseComponentPath, 'src');
+
+    fs.copySync(baseComponentFiles, componentPath);
+
+    const componentPackage = require(componentPath + '/package.json');
```

Comment:
I would use readJsonSync as you are requiring something and then overriding its own cache by changing the file immediately after...